aws-s3-source.kamelet.yaml

```diff
@@ -70,5 +81,7 @@ spec:
         accessKey: "{{accessKey}}"
         region: "{{region}}"
         deleteAfterRead: "{{deleteAfterRead}}"
+        uriEndpointOverride: "{{uriEndpointOverride}}"
```

Comment:
I think it's better to make this parameter optional. I mean, it will work anyway, but we're going to have it filled with an empty string. Nothing critical.

Reply:
good point! so the idea would be to provide this parameter via Camel message header in a choice when set, right?

Reply:
that's better! thx